### ISSUE TITLE
[BUG] docs.rs builds are failing for holochain_conductor_api 0.2.6

### ISSUE THREAD

**Issue by steveej** [holochain/holochain#3518]
see the [most recent build log](https://docs.rs/crate/holochain_conductor_api/0.2.6/builds/1117554)

this is the crux of it

```
[INFO] [stderr] error[E0635]: unknown feature `stdsimd`
[INFO] [stderr]   --> /opt/rustwide/cargo-home/registry/src/index.crates.io-6f17d22bba15001f/ahash-0.7.7/src/lib.rs:33:42
[INFO] [stderr]    |
[INFO] [stderr] 33 | #![cfg_attr(feature = "stdsimd", feature(stdsimd))]
[INFO] [stderr]    |                                          ^^^^^^^
```

* related [forum thread](https://users.rust-lang.org/t/error-e0635-unknown-feature-stdsimd/106445).
* [aHash upstream issue](https://github.com/tkaitchuck/aHash/issues/200)

it's probably enough to update our ahash dependency

**Comment by ThetaSinner** [comment]
Last time I looked, `ahash` hadn't yet released an updated dependency with this fixed. Is that the case now?

**Comment by ThetaSinner** [comment]
This is actually already fixed on the 0.2 branch. I just checked that builds and docs are okay on the latest nightly. You can also see that the RC releases for 0.2.7 are working -> https://docs.rs/crate/hdk/0.2.7-rc.1

I'm going to close this as there's nothing more for us to do on this issue. The problem will be solved when 0.2.7 releases.
